### PR TITLE
Top nav

### PR DESCRIPTION
**Pull request by dinilandy**
1. Utilize an array to store page names and corresponding routings dynamically.
2. Ensure the navigation component supports easy addition and removal of pages without hardcoding.
3. Utilize Material-UI (MUI) components for the design and styling of the navigation component.

### CODE REVIEW

**Comment by dinilandy** [comment]
Issues #12

**Comment by dinilandy** [comment]
I wasn't sure if I should create a new CSS style file, so I added the styles directly within the component.

**Comment by DvoraShechter1** [comment]
Code looks good! Some minor comments:

1. In the file src/components/Nav.jsx -
    Delete the empty line in the array (11)
    Use the appropriate MUI component for the image and not the img tag, that way you won't need internal styling.
2. Changes to the package-lock file should be undone.
3. A link to the issue should be in the description and not in a separate comment.

good luck!

**Comment by dinilandy** [comment]
I made the changes.

**Comment by DvoraShechter1** [comment]
The changes look perfect!
Pay attention to points 2 and 3 in the previous comment.

**Comment by dinilandy** [comment]
I saw this .
Thanks

File: package-lock.json [hunk 1]
```diff
@@ -2446,7 +2446,6 @@
       "version": "0.18.0",
       "resolved": "https://registry.npmjs.org/@eslint/config-array/-/config-array-0.18.0.tgz",
       "integrity": "sha512-fTxvnS1sRMu3+JjXwJG0j/i4RT9u4qJ+lqS/yCGap4lH4zZGzQ7tu+xZqQmcMZq5OBZDL4QRxQzRjkWcGt8IVw==",
-      "dev": true,
```
Comment (DvoraShechter1) [inline package-lock.json]: you need cancel this line, and others line like it.

**Review by DvoraShechter1 — APPROVED** [review]
LGTM!

**Pull request merged**